dashboard/pubspec.yaml

```diff
@@ -30,6 +30,7 @@ dependencies:
   protobuf: ^2.0.1
   provider: ^6.0.2
   url_launcher: ^6.0.6
+  icon_plugin: ^0.0.2
```

Comment:
I think calling it icon is going to cause confusion. See https://api.flutter.dev/flutter/material/Icons-class.html where the namespace is dedicated for font like icons.

Maybe flutter_app_icons would give better visibility in the pub.dev ecosystem?

Reply:
Just to follow up, I do appreciate you writing the plugin for this! This is great, and I'm happy to see this functionality get added to the ecosystem

Reply:
:) I won't take it badly even if the pull request doesn't land. I used flutter for a small project, and I'll be using it for a much bigger project. So trying to give back as much as I can.

Reply:
> I think calling it icon is going to cause confusion. See https://api.flutter.dev/flutter/material/Icons-class.html where the namespace is dedicated for font like icons.
> 
> Maybe flutter_app_icons would give better visibility in the pub.dev ecosystem?

True concerning the naming. I'll change it. Can you clue me concerning the failing tests too pls?